docs/Node Operators/fulfilling-requests.md

```diff
@@ -4,384 +4,288 @@ date: Last Modified
 title: "Fulfilling Requests"
 permalink: "docs/fulfilling-requests/"
 whatsnext: {"Performing System Maintenance":"/docs/performing-system-maintenance/", "Miscellaneous":"/docs/miscellaneous/", "Best Security and Operating Practices":"/docs/best-security-practices/"}
-metadata: 
+metadata:
   title: "Chainlink Node Operators: Fulfilling Requests"
   description: "Deploy your own Oracle contract and add jobs to your node so that it can provide data to smart contracts."
 ---
 With your own Oracle contract, you can use your own node to fulfill requests. This guide will show you how to deploy your own Oracle contract and add jobs to your node so that it can provide data to smart contracts.
-[block:api-header]
-{
-  "title": "Prerequisites"
-}
-[/block]
-- Going through the [Beginner Walkthrough](../beginners-tutorial) will help you obtain Testnet LINK and set up Metamask- [Run an Ethereum Client](../run-an-ethereum-client/) - [Running a Chainlink Node](../running-a-chainlink-node/)
-[block:callout]
-{
-  "type": "success",
-  "title": "Make sure to fund your node address!",
-  "body": "In the `keys` tab, you'll see at the bottom `Account Addresses`. The address of your node is the `regular` one. \n\nIN OLDER VERSIONS: Go to `configuration` in your node. You'll see `ACCOUNT_ADDRESS`. This is the address of your node. Send this address ETH. You can find testnet ETH on various [faucets](../link-token-contracts/).\n\nIf you don't see `ACCOUNT_ADDRESS` there, check the 'Keys' tab and scroll down"
-}
-[/block]
 
-[block:api-header]
-{
-  "title": "Deploy your own Oracle contract"
-}
-[/block]
-- Go to <a href="https://remix.ethereum.org/#gist=03a079b9055f42d993d0066d6f454c6f&optimize=true&version=soljson-v0.4.24+commit.e67f0147.js" target="_blank" rel="noreferrer, noopener">Remix</a> and expand the gist menu
+##Requirements
```

Comment:
This renders fine, but could we add a space in between the hashes and the text for conformity with the rest of the page, please?

```suggestion
## Requirements
```